cmd/mattermost/commands/config.go

```diff
@@ -161,17 +163,34 @@ func configGetCmdF(command *cobra.Command, args []string) error {
 }
 
 func configShowCmdF(command *cobra.Command, args []string) error {
-	configStore, err := getConfigStore(command)
+	useJSON, err := command.Flags().GetBool("json")
 	if err != nil {
-		return err
+		return errors.Wrap(err, "failed reading json parameter")
 	}
 
 	err = cobra.NoArgs(command, args)
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("%s", prettyPrintStruct(*configStore.Get()))
+	configStore, err := getConfigStore(command)
+	if err != nil {
+		return err
+	}
+
+	config := *configStore.Get()
+
+	if useJSON {
+		configJSON, err := json.Marshal(config)
```

Comment:
Maybe, as far as we are using the shell, makes sense to use MarshalIndent here to allow the user to easily read the output anyway.